### ISSUE TITLE
[request] filter downloadable title list

### ISSUE THREAD

**Issue by ODA-AG** [V10lator/NUSspli#257]
The list of titles is long and I would like the ability to filter the following:

1. Region (for me just filter out the Japan and European titles)
2. Already installed (filter out titles that are already installed)
3. Platform (filter out Wii vs Wii U vs GBA, etc...)

This would make it much quicker to download the titles I am looking for without having to go through the whole list.

**Comment by Ducky-Duck-Duck** [comment]
I agree, I think the filter list in generally alright but it could most definitely be better. Also a way to change sorting would be great. Something more like pkgifor the ps3 
![Screenshot_20231203-171852](https://github.com/V10lator/NUSspli/assets/111944282/1071ff55-78b4-4c5d-b816-64d0dfd220bf)
